storybook/stories/NotificationStories.re

```diff
@@ -0,0 +1,27 @@
+open BsReactNative;
+
+Storybook.(
+  storiesOf("Notification", module_)
+  |> add("default", () => <Notification />)
+  |> add("notification share", () =>
+       <View style=Style.(style([flex(1.)]))>
+         <NotificationShare
+           title={j|Notificação 012345|j}
+           eventId="AKDJF19939399291DKD"
+           date={Js.Date.make() |> Js.Date.toString}
+           fields=[|
```

Comment:
we'll be actually the JSON here with the notification stuff right

Reply:
to treeviewer

Reply:
ahh, top. Thought it would be a "raw" of the notification content

Reply:
yea this way we would need to write a json parser from JSON to the Field variant

Reply:
`What` and `When` could be the way its and the treeviewer already opened could be the `More info`in this structure

Reply:
what you think, guys?
![image](https://user-images.githubusercontent.com/8146889/54557658-76e4ad00-499a-11e9-8121-ed8e9d7f0a40.png)